contrib/terraform-testing-infrastructure/modules/config-files/templates/initialize_accumulo.sh.tftpl

```diff
@@ -22,7 +22,7 @@ docker run -d --name jaeger \
     -p 6832:6832/udp -p 5778:5778 \
     -p 16686:16686 -p 14268:14268 \
     -p 14250:14250 -p 9411:9411 \
-    jaegertracing/all-in-one:1.29
+    artifactory.adv.evoforge.org/docker-remote/jaegertracing/all-in-one:1.29
```

Comment:
I don't believe this is a publicly accessible image repository. At least, I'm not able to pull from it.
```bash
$ docker pull artifactory.adv.evoforge.org/docker-remote/jaegertracing/all-in-one:1.29
Error response from daemon: Get "https://artifactory.adv.evoforge.org/v2/": Failed to lookup host: artifactory.adv.evoforge.org
```

Reply:
Thx, side effect of `git commit -a`.

Reply:
You could push a repo name through as a variable (default it to docker.io) if you want to be able to change where these images are pulled from w/o having to modify the source code.